metricbeat/module/kubernetes/state_container/_meta/testdata/kube-state-metrics.v1.3.0.plain-expected.json

```diff
@@ -0,0 +1,549 @@
+[
+    {
+        "event": {
+            "dataset": "kubernetes.container",
+            "duration": 115000,
+            "module": "kubernetes"
+        },
+        "kubernetes": {
+            "container": {
+                "_module": {
```

Comment:
All these fields also have to go onto the module level.

Reply:
Placed on to the module (not root) level